CHANGELOG.md

```diff
@@ -115,4 +118,4 @@
 
 -   Upgrade dbt-core to 1.3.2.
 
--   Upgrade dbt-tests-adapter to 1.3.2.
+-   Upgrade dbt-tests-adapter to 1.3.2.
```

Comment:
Could you please keep the empty line in the end of the file? It should be a parameter in your IDE somewhere.